Src/Support/Google.Apis.Auth/OAuth2/UserCredential.cs

```diff
@@ -80,6 +80,12 @@ public UserCredential(IAuthorizationCodeFlow flow, string userId, TokenResponse
         IGoogleCredential IGoogleCredential.WithQuotaProject(string quotaProject) =>
             new UserCredential(Flow, UserId, Token, quotaProject);
 
+        /// <inheritdoc/>
+        IGoogleCredential IGoogleCredential.WithHtppClientFactory(IHttpClientFactory httpClientFactory) =>
+            Flow is IHttpAuthorizationFlow httpFlow ?
+            new UserCredential(httpFlow.WithHttpClientFactory(httpClientFactory), UserId, Token, QuotaProject) :
```

Comment:
Nit: I tend to put the `?` and `:` at the start of lines, as then it makes it clearer how they relate to the condition.